src/utils/setup/stride.rs

```diff
@@ -0,0 +1,70 @@
+use localic_std::{errors::LocalError, transactions::ChainRequestBuilder};
+use log::info;
+use serde_json::Value;
+
+use crate::{utils::test_context::TestContext, ADMIN_KEY, STRIDE_CHAIN_NAME};
+
+pub fn set_up_host_zone(test_ctx: &mut TestContext, dest_chain: &str) {
```

Comment:
This should probably be a member function of the TestContext for consistency with the rest of the API. Same for other functions.

Reply:
Done!